### PR TITLE
Agnostos tools

### PR DESCRIPTION
**Pull request by mschecht**
Hi Meren,

This PR covers the code necessary for integrating [AGNOSTOS](https://github.com/functional-dark-side/agnostos-wf) into anvio projects and visualization in the interactive interface.

I think we should merge this branch for the usability of the metagenomics applications in the [tutorial](https://merenlab.org/2021/05/28/agnostos/).

Thanks!

### CODE REVIEW

File: sandbox/anvi-script-add-unknowns-to-pangenome [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+from anvio.dbops import PanSuperclass
+
+args = argparse.Namespace(pan_db="Enterococcus_faecalis/Enterococcus_faecalis-PAN.db", genomes_storage="Enterococcus-faecalis-GENOMES.db")
+
+pan_db = PanSuperclass(args)
+
+pan_db.init_gene_clusters()
+pan_db.init_gene_clusters_functions()
+
+# print(pan_db.gene_clusters_functions_dict['GC_00000212'])
```
Comment (meren) [inline sandbox/anvi-script-add-unknowns-to-pangenome:34]: why is this here?

Comment (mschecht) [reply sandbox/anvi-script-add-unknowns-to-pangenome]: removed!
